src/t_zset.c

```diff
@@ -72,12 +72,41 @@ void zsetConvertAndExpand(robj *zobj, int encoding, unsigned long cap);
 zskiplistNode *zslGetElementByRankFromNode(zskiplistNode *start_node, int start_level, unsigned long rank);
 zskiplistNode *zslGetElementByRank(zskiplist *zsl, unsigned long rank);
 
+static inline unsigned long zslGetNodeSpanAtHeight(zskiplistNode *x, int height) {
+    return height > 0 ? x->level[height].span : x->level[height].forward ? 1
+                                                                         : 0;
```

Comment:
Nested `?:` is easier to read if it's indented similar to `else if` ...

```C
    return height > 0 ?
               x->level[height].span :
           x->level[height].forward ?
               1 :
           0;
```

... but I suppose clang-format doesn't like this style. Let's avoid the nested ?: and use `if` in one of the levels?

```suggestion
    if (height > 0) return x->level[height].span;
    return x->level[height].forward ? 1 : 0;
```

Reply:
> What can we call this the inverse lookup? It deserves a cool name.

RRC (Revers Rank Check)?